content/millennium/r4/individuals/related-person.md

```diff
@@ -73,8 +73,9 @@ _Implementation Notes_
 Notes:
 
 - When provided, the `identifier` query parameter must include both a system and a code. Example: `identifier=urn:oid:2.16.840.1.113883.3.13.6|RANDES011E8442-2E86-4A00-8394-EFC6A3434A8F`
-- `-relationship-level` will be required with patient and identifier searches in the future.
+- `-relationship-level` will be required with patient and identifier searches.
 - `-relationship-level` can only be provided with `patient` or `identifier`
+- `-relationship-level` can only be set to `Patient`
```

Comment:
Nit: Missing period in the end.

Reply:
We should either remove the period on 76or add them to the others.

Reply:
removed period for line 76 here 
https://github.com/cerner/fhir.cerner.com/pull/590/commits/e3709c4ee555f818baaa45357eb71e9ca2c828e8#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR76

Reply:
+1